Makefile

```diff
@@ -164,7 +164,7 @@ build/config.BUILT:
 build: build/llvm.BUILT build/wasi-libc.BUILT build/compiler-rt.BUILT build/libcxxabi.BUILT build/libcxx.BUILT build/config.BUILT
 
 strip: build/llvm.BUILT
-	cd $(PREFIX)/bin; strip clang-9 lld llvm-ar
```

Comment:
I would imagine that all the clang binaries are symbolic links to each other?  I wonder why this wasn't just "clang" from the very beginning?

Reply:
Should it actually be something like `find $(PREFIX)/bin -type f -executable -exec strip {} \;` to get all of the binaries?

Reply:
yes, something like that sounds better.